cypress/integration/Signup.spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+import * as signUp from '../page-objects/Signup';
+
+const { internet } = require('faker');
+
+describe('Signup tests', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.env('HOME_URL'));
+    signUp.signUpPage();
+  });
+
+  it('Should sign up succesfully', () => {
+    const id = Math.floor(Math.random() * 10000);
+    const username = `username${id}`;
+    signUp.addUsername(username); // internet.userName genera un usuario con fomato xxx.yyy y conduit no acepta ese formato
+    signUp.addEmail(internet.exampleEmail());
+    signUp.addPassword(internet.password());
+    signUp.clickSignUpButton();
+    signUp.getLoggedUsername(username);
```

Comment:
There should be more asserts here. Also, to check if the user was correctly signed up, you can try to login, this way the tests is more e2e, and not so unit test.